System/ContentFile.cpp

```diff
@@ -100,6 +106,93 @@ namespace RTE {
 		m_DataPathAndReaderPosition = m_DataPath + "\n" + newPosition;
 	}
 
+/////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////////
+
+	int ContentFile::GetImageFileInfo(ImageFileInfoType infoTypeToGet) {
+		bool fileInfoIsEmpty = false;
```

Comment:
I would rename this to fileInfoIsNotFull to better describe it.

Though I guess if fileinfo will never be partially filled, you could instead just do `bool fileInfoIsEmpty = m_ImageFileInfo[0] == -1;` right?

Reply:
Better be safe than sorry.

Reply:
Sure, then please rename for clarity.